src/goto-symex/symex_target_equation.cpp

```diff
@@ -318,6 +318,19 @@ void symex_target_equationt::constraint(
   merge_ireps(SSA_step);
 }
 
+void symex_target_equationt::convert_without_assertions(
```

Comment:
> Could this be refactored so that we don't have almost-the-same-code twice?

https://github.com/diffblue/cbmc/pull/4361#discussion_r292643973

Looks to be outstanding

Reply:
Refactored.